src/locales/en/validation.json

```diff
@@ -0,0 +1,23 @@
+{
+  "userName": {
+    "required": "Name is required!"
+  },
+
+  "email": {
+    "required": "Email is required!",
+    "invalidFormat": "Please enter a valid email format!"
+  },
+
+  "dateOfBirth": {
+    "required": "Birth date is required!"
+  },
+
+  "password": {
+    "required": "Password is required!",
+    "invalidFormat": "Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and One Special Case Character"
```

Comment:
Is it right, that all words start with uppercase?

Reply:
Edited the translation of that field